master-firmware/src/main.c

```diff
@@ -268,6 +268,15 @@ void init_base_motors(void)
     motor_manager_create_driver(&motor_manager, "right-wheel");
 }
 
+
+void init_hands(void)
+{
+    bus_enumerator_add_node(&bus_enumerator, "left-fingers", NULL);
+    bus_enumerator_add_node(&bus_enumerator, "left-sensors", NULL);
+    bus_enumerator_add_node(&bus_enumerator, "right-fingers", NULL);
```

Comment:
initially, we planned two io-boards per hand: one for controlling the fingers and one for the sensors